docs/monitoring/inventory.md

```diff
@@ -46,21 +43,61 @@ In addition, an overview of all the installed packages on a given system is avai
 - Package name
 - Package version
 
-![Inventory Overview](../images/monitoring/inventory_packages.png)
+#### Comparing Hosts
+
+The Inventory UI allows comparing different systems to quickly and efficiently see differences between them. Use filtering to see differences in specific packages.
+
+<video style="display:block; width:100%; height:auto;" controls autoplay loop>
+  <source src="https://cdn.sematext.com/videos/sematext-infra-inventory-hosts-compare.mp4" type="video/mp4" />
+</video>
+
+#### Grouping Hosts By Installed Packages
+
+Hosts can be filtered and grouped by tags. One of the available tags for grouping is installed.packagas tag, which will put servers with exactly the same packages installed in the same group. This can be useful to quickly check if all servers that should have the same packages installed are indeed the same.
+
+### Packages
+#### Info
 
-### Comparing Hosts
+Provides quick insight into how unified your packages are by showing:
 
-The Inventory UI allows comparing different systems to quickly and efficiently see differences between them:
+- Number of package types
+- Number of different packages
+- Number of package installations
+
+<video style="display:block; width:100%; height:auto;" controls autoplay loop>
+  <source src="https://cdn.sematext.com/videos/sematext-infra-inventory-packages.mp4" type="video/mp4" />
+</video>
+
+#### Overview
+
+The Packages overview provides a per-package view of all package installations across hosts. Find out how many installations of each package there are, where they are installed and in which version.
+
+### Container Images
+#### Info
+
+Provides quick insight into how unified your container images are by showing:
+
+- Number of different container images
+- Number of different container image versions
+- Total number of container images
+
+<video style="display:block; width:100%; height:auto;" controls autoplay loop>
+  <source src="https://cdn.sematext.com/videos/sematext-infra-inventory-container-images.mp4" type="video/mp4" />
+</video>
+
+#### Overview
+
+The Container Images overview provides a per-container-image view of all container images across hosts. Find out how many copies of each container image there are, where they are installed and in which verison.
```

Comment:
The Container Images overview provides a per-container-image view of all container images across hosts. Find out how many copies of each container image there are, where they are installed and in which verison.
=>
The Container Images overview provides a per-container and per-image view of all container images across hosts. Find out how many copies of each container image there are, where they are installed and which version they are tagged with.